packages/babel-helper-replace-supers/src/index.js

```diff
@@ -61,19 +61,9 @@ export const environmentVisitor = {
 const visitor = traverse.visitors.merge([
   environmentVisitor,
   {
-    ReturnStatement(path, state) {
-      // TODO get this shit out of here
```

Comment:
lol 🙌 ?

Reply:
https://github.com/babel/babel/pull/7737/files#diff-74b3eb66512735b6b5013e5d41e33642R65. I've been planing this removal for the issues linked in OP.